code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1320,6 +1323,107 @@ GLOBAL_LIST_EMPTY(features_by_species)
 
 	return TRUE
 
+#define HEADSMASH_BLOCK_ARMOR 20
+
+//TODO: Add a grab state check on the do_afters
+/datum/species/proc/try_grab_maneuver(mob/living/carbon/human/target, mob/living/carbon/human/user, modifiers)
+	if(!target)
+		return
+	var/obj/item/bodypart/affecting = target.get_bodypart(ran_zone(user.zone_selected))
+	if(!affecting)
+		return FALSE
+	. = FALSE
+	if(HAS_TRAIT(user, TRAIT_PACIFISM)) //They're mostly violent acts
+		return
```

Comment:
Please insert a self-message mentioning to pacifists that it is something they can't do.